web/css/json-editor.css

```diff
@@ -156,6 +156,9 @@ body { -webkit-animation: androidBugfix infinite 1s; }
 }
 
 /* Styles specific to the schema content */
+.content {
```

Comment:
@starryeyez024 Where is the class content coming from?  I don't see it in any of the templates so I'm having a hard time finding an example to test that this works.

Reply:
bump

Reply:
@starryeyez024 double bump

Reply:
I can't find this either. Sorry for the trouble, going to close this out.